src/storage.ts

```diff
@@ -435,8 +435,8 @@ export class Storage {
   /**
    * Configure the CLI for the deployment with the provided label.
    *
-   * Falsey values are a no-op; we avoid unconfiguring the CLI to avoid breaking
-   * existing connections.
+   * Falsey URLs and null tokens are a no-op; we avoid unconfiguring the CLI to
+   * avoid breaking existing connections.
    */
   public async configureCli(label: string, url: string | undefined, token: string | undefined | null) {
```

Comment:
We don't pass an `undefined` token to this function anywhere as far as I can tell, so I think we should remove it from the type signature to prevent a future headache where we think it should act like null but it actually acts like "", if that makes sense.

Honestly a rule around keeping undefined out of function parameter type signatures isn't a bad idea. For me undefined is for uninitialized values.

Reply:
Removed!  Makes sense to limit the types to the bare minimum we need.